mssqlcli/main.py

```diff
@@ -63,7 +63,10 @@ def configure_and_update_options(options):
         if not options.username:
             options.username = input(u'Username (press enter for sa):') or u'sa'
         if not options.password:
-            options.password = getpass.getpass()
+            pw = getpass.getpass()

```

Comment:
why is this breaking now? we ran on python 2/3 with this code as is.

Reply:
As I mentioned in the description, when I type `abcd` for password in **command prompt**and hit `ENTER`, `getpass.getpass()` returns `abcd\r`, and it is stored in `options.password`. This made login failed to db since it always have `\r` character for the `ENTER` key in Python2

Reply:
But getpass isn't prompt_tookit. Have your tried `pw = getpass.getpass().rstrip()`? Seems more concise and take care of newlines and whitespace.

Reply:
nevermind - I like your approach better since a space can be part of a password.